arch/x86/kernel/cpu/resctrl/core.c

```diff
@@ -602,7 +602,7 @@ static void clear_closid_rmid(int cpu)
 	state->default_rmid = 0;
 	state->cur_closid = 0;
 	state->cur_rmid = 0;
-	wrmsr(MSR_IA32_PQR_ASSOC, 0, 0);
+	wrmsr(MSR_IA32_PQR_ASSOC, 0, RESCTRL_RESERVED_CLOSID);
```

Comment:
At upstream they changed many of these zeroes with variables. (https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=6791e0ea30711b937d5cb6e2b17f59a2a2af5386)

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev